### ISSUE TITLE
Write model.md

### ISSUE THREAD

**Issue by marlitas** [phetsims/center-and-variability#443]
The model.md needs to be written for both the center-and-variability and soccer-common repos.

**Comment by samreid** [comment]
I wrote an initial draft, would be good for @marlitas and @matthew-blackman to see what else should be added.

**Comment by pixelzoom** [comment]
For code review #447, I made these changes in the above commits:

* Added TODO comments where text seemed missing.
* Fixed some misc grammar and punctuation.
* Changed "IQR (interquartile):" to "IQR (interquartile range):"

**Comment by pixelzoom** [comment]
In soccer-common/doc/model.md, this sentence is not very information, and sort of confusing.  It's not clear to me how a soccer ball has a value, or what value you're referring to.  It will probably be even less clear to the audience for this doc.

> The soccer ball has a null value until it lands, at which time it takes a numerical value.

**Comment by samreid** [comment]
Looks good to me, @pixelzoom what do you think?

**Comment by pixelzoom** [comment]
Looks good to me!  

model.md sign-off is the responsibility of the sim designer, so that's the next review step.

**Comment by samreid** [comment]
Thanks, @catherinecarter can you please review both:

https://github.com/phetsims/center-and-variability/blob/main/doc/model.md
https://github.com/phetsims/soccer-common/blob/main/doc/model.md

You can commit changes to them directly or request a developer to do so. Thanks!

**Comment by catherinecarter** [comment]
I looked at (and edited) the model.md file in the center-and-variability repo. I made some changes and added details to the TODOs @pixelzoom flagged in the document. @amanda-phet, if you want to take a last look to edit/review, that would be great.

For the model.md file in soccer-common, I have a question.
- Is there a reason why the words "kickers" and "soccer balls" are italicized the document? Example (in the 2nd paragraph):

   >The Soccer Common repository is used to create Soccer Scenes in which one or more _kickers_ can kick _soccer balls_, which land along a horizontal number line and stack vertically. The main functionalities of the model include:
